search/lib/api/provider.js

```diff
@@ -71,12 +151,45 @@ async function getProviders (request, response) {
   response.status(200).json(providerCatalog);
 }
 
+/**
+ * Get all the providers.
+ *   Note: We can't query cloud holding providers, so this will return all the providers.
+ *         But it will return the links to cloud holding collections for these providers.
+ *         It is possible that a link returns nothing if that provider doesn't contain cloud holding collections.
+ */
+async function getCloudProviders (request, response) {
+  const event = request.apiGateway.event;
+  const providers = await cmr.getProviderList();
+  const providerCloudLinks = await Promise.map(providers, async (provider) => {
+    return {
+      title: provider['short-name'],
+      rel: 'child',
+      type: 'application/json',
+      href: generateCloudAppUrl(event, `/${provider['provider-id']}`)
```

Comment:
So is this basically the only difference between this function and getProviders?  The creation of the child links?

Reply:
And title, description.